storage/mdadm/include/PURPOSE

```diff
@@ -0,0 +1,6 @@
+Please check http://wiki.test.redhat.com/fge/mdadm_include
```

Comment:
This page doesn't seem to exist when I tried to access it. Can instead the information needed (unless including customer data) be added to this file instead of being linked too?

Reply:
Also we don't want to include internal hostnames in open sourced tests :-)

Reply:
I have delete unnecessary files from common (include renamed common)